lib/src/svg/colors.dart

```diff
@@ -36,21 +36,27 @@ Color parseColor(String colorString) {
 
   // handle rgb() colors e.g. rgb(255, 255, 255)
   if (colorString.toLowerCase().startsWith('rgb')) {
-    final List<int> rgb = colorString
+    final List<num> rgb = colorString
```

Comment:
I think we should probably keep this as a `List<int>` - things could go very wrong if we actually pass a double to the `Color` constructors that expect an `int`, not a `double`.

Reply:
Oh!  Only the opacity is allowed to be a double.